File: JS/61-70/e_62.js
function filtraPalabras(frase, longitudMinima) {
    // Dividir la frase en palabras
    var palabras = frase.split(" ");
    
    // Filtrar las palabras según la longitud mínima
    var palabrasFiltradas = palabras.filter(function(palabra) {
        return palabra.length >= longitudMinima;
    });
    
    // Reconstruir la frase con las palabras filtradas
    var fraseFiltrada = palabrasFiltradas.join(" ");
    
    return fraseFiltrada;
}

// Ejemplo de uso
var fraseOriginal = "Esta es una frase de ejemplo para filtrar palabras según su longitud";
var longitudMinima = 4;
var fraseFiltrada = filtraPalabras(fraseOriginal, longitudMinima);
console.log("Frase original:", fraseOriginal);
console.log("Frase filtrada:", fraseFiltrada);


// Esta función filtraPalabras toma la frase original y la longitud mínima como parámetros. 
// Divide la frase en palabras, filtra las palabras según la longitud mínima especificada y luego reconstruye la frase con las palabras filtradas.

File: Python/61-70/e_62.py
# Escribe una función en Python  llamada filtraPalabras(frase,longitudMinima), 
# que tome como parámetros una frase y filtre las palabras con una longitud menor al establecido en el parámetro longitudMinima. 
# La función debe devolver la misma frase sin las palabras filtradas.


def filtraPalabras(frase, longitudMinima):
    # Dividir la frase en palabras
    palabras = frase.split()
    
    # Filtrar las palabras según la longitud mínima
    palabras_filtradas = [palabra for palabra in palabras if len(palabra) >= longitudMinima]
    
    # Reconstruir la frase con las palabras filtradas
    frase_filtrada = ' '.join(palabras_filtradas)
    
    return frase_filtrada

# Ejemplo de uso
frase_original = "Esta es una frase de ejemplo para filtrar palabras según su longitud"
longitud_minima = 4
frase_filtrada = filtraPalabras(frase_original, longitud_minima)
print("Frase original:", frase_original)
print("Frase filtrada:", frase_filtrada)


# Esta función tomará la frase original y la longitud mínima como parámetros. Dividirá la frase en palabras, 
# luego filtrará las palabras según la longitud mínima especificada y finalmente reconstruirá la frase con las palabras filtradas.